Modules/_hashopenssl.c

```diff
@@ -74,6 +74,12 @@
 #define PY_EVP_MD_CTX_md(CTX)   EVP_MD_CTX_md(CTX)
 #endif
 
+static inline int
+PY_EVP_MD_xof(PY_EVP_MD *md)
```

Comment:
This one is needed for API compatibility (it doesn't exist in 1.1.1 and it's only in 3.5 or something)